src/js/setUpSite.ts

```diff
@@ -160,6 +170,17 @@ const setScorecard = (cityId: CityId, cityProperties: ScoreCard): void => {
     autoPan: false,
   }).setContent(scorecard);
   layer.bindPopup(popup).openPopup();
+
+  const alertBox = document.getElementById(
+    "community-map-alert"
+  ) as HTMLElement;
+  if ("contribution" in details) {
+    const email = document.getElementById("email") as HTMLSpanElement;
+    email.innerText = details.contribution;
+    alertBox.style.display = "block";
+  } else {
+    alertBox.style.display = "none";
```

Comment:
This should also be set as the initial CSS so that we don't have a flicker when you first load the site.

Reply:
Yup, I like that idea better. Scorecard has important information. The alert is importation information.